image_transport/test/utils.hpp

```diff
@@ -22,17 +22,32 @@
 
 #include "gtest/gtest.h"
 
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/rclcpp.hpp>
+#include <rclcpp/node_interfaces/node_interfaces.hpp>
 
 namespace test_rclcpp
 {
+using NodeGraphInterface = rclcpp::node_interfaces::NodeGraphInterface;
+using NodeLoggingInterface = rclcpp::node_interfaces::NodeLoggingInterface;
+using NodeBaseInterface = rclcpp::node_interfaces::NodeBaseInterface;
+using NodeParametersInterface = rclcpp::node_interfaces::NodeParametersInterface;
+using NodeTopicsInterface = rclcpp::node_interfaces::NodeTopicsInterface;
+using NodeTimersInterface = rclcpp::node_interfaces::NodeTimersInterface;
+
+using RequiredInterfacesTest = rclcpp::node_interfaces::NodeInterfaces<
+  NodeBaseInterface,
+  NodeGraphInterface,
+  NodeLoggingInterface,
+  NodeParametersInterface,
+  NodeTimersInterface,
+  NodeTopicsInterface>;
```

Comment:
```suggestion
using NodeBaseInterface = rclcpp::node_interfaces::NodeBaseInterface;
using NodeGraphInterface = rclcpp::node_interfaces::NodeGraphInterface;
using NodeLoggingInterface = rclcpp::node_interfaces::NodeLoggingInterface;
using NodeParametersInterface = rclcpp::node_interfaces::NodeParametersInterface;
using NodeTimersInterface = rclcpp::node_interfaces::NodeTimersInterface;
using NodeTopicsInterface = rclcpp::node_interfaces::NodeTopicsInterface;

using RequiredInterfacesTest = rclcpp::node_interfaces::NodeInterfaces<
  NodeBaseInterface,
  NodeGraphInterface,
  NodeLoggingInterface,
  NodeParametersInterface,
  NodeTimersInterface,
  NodeTopicsInterface>;
```

This isn't really a coding standards comment, but having the order match that in the usage on the last line (Which also happens to be alphabetical) helps with readability/maintainability.